Source/OxyPlot.Wpf/Plot.cs

```diff
@@ -153,6 +153,24 @@ private static void AppearanceChanged(DependencyObject d, DependencyPropertyChan
         }
 
         /// <summary>
+        /// Called when the tool tip is changed.
+        /// </summary>
+        /// <param name="d">The d.</param>
+        /// <param name="e">The <see cref="System.Windows.DependencyPropertyChangedEventArgs" /> instance containing the event data.</param>
+        private static void TitleToolTipChanged(DependencyObject d, DependencyPropertyChangedEventArgs e)
+        {
+            var plot = (Plot)d;
+
+            var model = plot.ActualModel;
+            if (model != null)
+            {
+                model.TitleToolTip = e.NewValue as string;
```

Comment:
For other properties this is handled in `OnAppearanceChanged`. Why the different handling?